components/widgets/forest-change/integrated-deforestation-alerts/index.js

```diff
@@ -158,13 +160,23 @@ export default {
     // extract relevant metadata
     const defaultStartDate = GLAD?.defaultStartDate;
     const defaultEndDate = GLAD?.defaultEndDate;
-    const startDate = params?.startDate || defaultStartDate;
+    const selectedDate = params?.startDate || defaultStartDate;
     const endDate = params?.endDate || defaultEndDate;
 
     const isAoi = params?.locationType === 'aoi';
     const status = params?.status || 'unsaved';
     const isAnalysis = shouldQueryPrecomputedTables(params);
 
+    // overriding start date (FLAG-593)
```

Comment:
I think for the same file you can use a method to avoid duplication.

`const setStartDateByAlertSystem = (alertSystem, params, selectedDate = null) => { ... }`